subscribie/blueprints/admin/__init__.py

```diff
@@ -1899,6 +1899,28 @@ def vat_settings():
     return render_template("admin/settings/vat_settings.html", settings=settings)
 
 
+@admin.route("/donate-enabled-settings", methods=["GET", "POST"])
+@login_required
+def donations_enabled_settings():
+    settings = Setting.query.first()  # Get current shop settings
+
+    if settings is None:
```

Comment:
Good defensive programming here.

It's now safe to assume `settings` will alway exist and the check can be deleted since we automatically setup settings in one during shop creation.